content/releases.md

```diff
@@ -1,4 +1,69 @@
-## v1.1.0 - "Training Wheels"
+## Release 1.2.0 - "Kubernetes For The Edge"
+
+[Get Started with release 1.2.0](/docs/1.1.0/getting-started/core-concepts.html)
+
+Large ioFog release brings a set of features for seamless integration of Kubernetes and Edge Computing. The release
+comes with a new command line interface for Edge Compute Network management called _iofogctl_ and other supplementary
+tools, such as _platform tools_ for cluster and infrastructure management.
+
+### What's New?
+
+- **Kubernetes integration!** First release of ioFog supporting Kubernetes integration, and first releases of [iofog-kubelet](https://github.com/eclipse-iofog/iofog-kubelet/releases/tag/v1.2.0) and [iofog-operator](https://github.com/eclipse-iofog/iofog-operator/releases/tag/v1.2.0) microservices
+- **[Iofogctl](https://github.com/eclipse-iofog/iofogctl/releases/tag/v1.2.0)** command line interface for management of Edge Compute Networks (see [iofogctl tutorial](/docs/1.2.0/tools/iofogctl.html)) running on both Linux and Mac
+- **[Helm chart](https://github.com/eclipse-iofog/helm/releases/tag/v1.2.0)** for easy installation of ioFog Edge Compute Network to existing Kubernetes cluster (see [helm tutorial](/docs/1.2.0/getting-started/how-to-helm.html))
+- **[Platform tools](https://github.com/eclipse-iofog/platform/tree/1.2.0)** for easy infrastructure and Kubernetes cluster setup on Google Cloud Platform and Packet (see [platform tutorial](/docs/1.2.0/tools/platform-tools.html))
+- Many bugfixes in the ioFog engine: [controller](https://github.com/eclipse-iofog/Controller/releases/tag/v1.2.0), [connector](https://github.com/eclipse-iofog/Connector/releases/tag/v1.2.0) and [agent](https://github.com/eclipse-iofog/Agent/releases/tag/v1.2.0)
+
+### Changelogs
+
+##### Agent (v1.2.0)
+
+##### Connector (v1.2.0)
+
+##### Controller (v1.2.0)
+
+##### Iofogctl (v1.2.0)
```

Comment:
* Redesign output formats for success and error
* Refactor parallel agent installs to be thread-safe
* Require agents to be deleted before being redeployed
* Support relative paths for input files
* Add ~/.iofog directory for config files
* Add uptime metrics for get command
* Fix Kubernetes SSH tunnel bug
* Fix Kubernetes Connector ports bug